frontends/web/src/locales/en/app.json

```diff
@@ -372,13 +373,12 @@
           },
           "title": "Payment methods and fees"
         },
-        "privacyPolicy": "MoonPay privacy policy",
         "protection": {
-          "description": "The BitBoxApp does not collect any data when buying {{name}}, the incoming funds are treated like a regular transaction. MoonPay needs to collect some personal data to operate. Their Privacy Policy explains in detail how that data is handled.",
+          "description": "The BitBoxApp does not collect any data when buying {{name}}, the incoming funds are treated like a regular transaction. However MoonPay and Pocket need to collect some information to operate. Please refer to their respective privacy policies to see in more detail how the data is handled.",
           "title": "Data protection"
         },
         "security": {
-          "description": "When you buy {{name}} via MoonPay, you are using an external service. This service is out of scope of the BitBox02 Security Threat model and relies on the safety and security of the environment which the BitBoxApp software is running in.",
+          "description": "When you buy {{name}} via MoonPay and/or Pocket, you are using an external service. This service is out of scope of the BitBox02 Security Threat model and relies on the safety and security of the environment which the BitBoxApp software is running in.",
```

Comment:
since you are here anyway, please lowercase "Security Threat model"

Reply:
One more thing, updating existing text should be done in locize. but keep it here for now so we see what should change and can discuss the change.

once PR is accepted we can change locize.

Reply:
I changed "Security Threat model", but restored the rest of the text as it was in the beginning. Made a new `descriptionGeneric` with a generic text that avoid mentioning single exchanges, which is included in the guide.